modules/update-service-overview.adoc

```diff
@@ -9,58 +9,34 @@
 // * updating/updating-disconnected-cluster.adoc
 
 [id="update-service-overview_{context}"]
-= About the {product-title} update service
+= About the OpenShift Update Service
 
-The {product-title} update service is the hosted service that provides over-the-air
-updates to both {product-title} and {op-system-first}. It provides a graph,
-or diagram that contain _vertices_ and the _edges_ that connect them, of
-component Operators. The edges in the graph show which versions you can safely
-update to, and the vertices are update payloads that specify the intended state
-of the managed cluster components.
+The OpenShift Update Service (OSUS) is the service that provides over-the-air updates to both {product-title} and {op-system-first}. It provides a graph, or diagram, that contains _vertices_ and the _edges_ that connect them, of component Operators. The edges in the graph show which versions you can safely update to, and the vertices are update payloads that specify the intended state of the managed cluster components.
```

Comment:
The section does not indicate what kind of cluster is recommended to use the OSUS. If it is designed for disconnected clusters, I would suggest to add a statement or a note to express it.

Reply:
Adding towards end of section:
`For clusters with internet accessibility, Red Hat provides over-the-air updates through an OpenShift Container Platform update service as a hosted service located behind public APIs. However, for clusters in a restricted network, there’s no way to access these public APIs to obtain this information. In order to provide a similar user upgrade experience in a restricted network users can install the OpenShift Update Service locally, i.e. accessible within their disconnected network, as described in the following sections.`